plugins/kubecarrier.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubecarrier
+spec:
+  description: KubeCarrier is an open source system for managing applications and
+    services across multiple Kubernetes Clusters; providing a framework to centralize
+    the management of services and provide these services with external users in a
+    self service catalog.
```

Comment:
This is not a pragmatic description. It's a good marketing intro. If you can be more specific and direct that helps users, that's appreciated. PTAL how other plugins do this.

> KubeCarrier is an open source system for managing applications and
    services across multiple Kubernetes Clusters;

That's what KubeCarrier is, but "what does this kubectl plugin do"?

Reply:
@nmiculinic AFAICT your plugin description did not change.